src/docs/_assets/js/ruiSwatch.js

```diff
@@ -0,0 +1,39 @@
+class RuiSwatch extends HTMLElement {
```

Comment:
Please @adamkudrna to review those styles. It seems a little wild to me.

Reply:
The styles come from an `*.scss` file that was imported in the original React component that was used in Docz.

The styles *here* could become part of the documentation CSS file (`src/docs/_assets/stylesheets/extra.css`). This would work for `<rui-swatch>` as it is never used inside a shadow DOM. `<rui-icon>` however is used inside shadow DOM, se the style can not be in the documentation CSS file. I wanted to have a unified approach.

Reply:
I understand, we can tackle this anytime later, I think.